src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -62,16 +68,23 @@
 public class SkroutzGreekStemmer {
   public final static String DEFAULT_STOPWORD_FILE = "stopwords.txt";
   private final CharArraySet stopwords;
+  private Map<String, char[]> stepZeroExceptions;
 
   public SkroutzGreekStemmer(final CharArraySet stopwords) {
     this.stopwords = stopwords;
+    this.stepZeroExceptions = new HashMap(0);
   }
 
   public SkroutzGreekStemmer() {
     this.stopwords = SkroutzGreekStemmer.getDefaultStopSet();
+    try {
+      this.stepZeroExceptions = loadExceptions("step_0_exceptions");
+    } catch(Exception ex) {
+        this.stepZeroExceptions =  new HashMap(0);
```

Comment:
Do log the exception thrown, otherwise if something goes wrong, you'll never find it out :-)